frontend/src/features/codes/IndicesList.tsx

```diff
@@ -150,96 +153,95 @@ const IndicesList = (): JSX.Element => {
                 setCurrentPage={setCurrentPage}
                 indexType={currentIndexType}
                 setFormData={setFormData}
-                setCreateDialogOpen={setCreateDialogOpen}
+                setCreateDialogOpen={setIsModalOpen}
             />
-            <div className={"index-actions"}>
+            <div className="index-actions">
                 <Button
                     theme="black"
                     iconLeft={<IconPlus />}
-                    onClick={() => setCreateDialogOpen(true)}
+                    onClick={() => setIsModalOpen(true)}
                 >
                     Lisää/päivitä indeksi
                 </Button>
             </div>
             <EditIndexDialog
-                editDialogOpen={editDialogOpen}
-                closeDialog={closeDialog}
                 indexType={currentIndexType}
                 formData={formData}
                 setFormData={setFormData}
+                isModalOpen={isModalOpen}
+                closeModal={closeModal}
             />
         </div>
     );
 };
 
-const EditIndexDialog = ({indexType, formData, setFormData, editDialogOpen, closeDialog}) => {
-    const [saveIndex, {data: saveData, error: saveError, isLoading: isSaving}] = useSaveIndexMutation();
+const EditIndexDialog = ({indexType, formData, setFormData, isModalOpen, closeModal}) => {
```

Comment:
If all other references to this Dialog are renamed to use a Modal nomenclature, perhaps this function should be called EditIndexModal, too?

Reply:
Renamed :)